xml/System.Net/CookieContainer.xml

```diff
@@ -79,6 +79,19 @@
 
  ]]></format>
     </remarks>
+    <threadsafe>
```

Comment:
Is this a legit tag?

Reply:
[Yes](https://github.com/search?q=repo%3Adotnet%2Fdotnet-api-docs+threadsafe+language%3AXML&type=code&l=XML&p=1), but looks like it cannot handle embedded `<format>` :( Will move the content to `<remarks>`.